source/iterators.tex

```diff
@@ -333,6 +333,40 @@
     constexpr insert_iterator<Container>
       inserter(Container& x, ranges::iterator_t<Container> i);
 
+  // \ref{const.iterators}, constant iterators and sentinels
+  template<@\libconcept{indirectly_readable}@ I>
+    using iter_const_reference_t = @\seebelow@;           // freestanding
```

Comment:
https://github.com/cplusplus/draft/pull/5713 seems to put the `// freestanding` comment as far right as possible.

Reply:
Noted, but this probably needs cleanup after all merges are done.